app/frontend/courses/students/components/CoursesStudents__StudentOverlay.res

```diff
@@ -246,6 +246,50 @@ let quizPerformanceChart = (averageQuizScore, quizzesAttempted) =>
   | None => React.null
   }
 
+let milestoneTargetsCompletionStats = studentDetails => {
+  let milestoneTargets = studentDetails->StudentDetails.milestoneTargetsCompletionStatus
+
+  let totalMilestoneTargets = Js.Array2.length(milestoneTargets)
+
+  let completedMilestoneTargets =
+    milestoneTargets->Js.Array2.filter(target => target.completed == true)->Js.Array2.length
+
+  let milestoneTargetCompletionPercentage = string_of_int(
+    int_of_float(
+      float_of_int(completedMilestoneTargets) /. float_of_int(totalMilestoneTargets) *. 100.0,
+    ),
+  )
+
+  <div className="flex items-center space-x-4">
+    <p className="text-xs font-medium text-gray-500 flex whitespace-nowrap">
```

Comment:
@vinutv 

Could you please help me style this(title, milestone_donut_chart and its div) better for mobile and tablet devices? 


![image](https://github.com/pupilfirst/pupilfirst/assets/53794102/5495de48-f11e-4867-8579-549bfbf7dd2d)

![image](https://github.com/pupilfirst/pupilfirst/assets/53794102/5c0318c5-7fa3-40ec-a1a9-2cc96a050334)

Reply:
@vinutv You can review and fix this from the main working branch after I merge this into it.